atomic_reactor/plugins/build_orchestrate_build.py

```diff
@@ -265,6 +272,104 @@ def __init__(self, tasker, workflow, platforms, build_kwargs,
         self.build_kwargs = build_kwargs
         self.osbs_client_config = osbs_client_config
         self.config_kwargs = config_kwargs or {}
+
+        self.build_kwargs['arrangement_version'] =\
```

Comment:
Let's create methods: `adjust_build_kwargs` and `adjust_config_kwargs`

Reply:
ok